fix cmake code for cross-build usage.
Initial directive "command KF5::desktoptojson ..." points
the target's arch desktoptojson binary instead of the host's one.

--- a/KF5CoreAddonsMacros.cmake
+++ b/KF5CoreAddonsMacros.cmake
@@ -45,7 +45,8 @@
         _desktop_to_json_cmake28(${desktop} ${json} ${DESKTOP_TO_JSON_COMPAT_MODE})
         return()
     endif()
-    set(command KF5::desktoptojson -i ${desktop} -o ${json})
+    get_target_property(DESKTOPTOJSON_LOCATION KF5::desktoptojson LOCATION)
+    set(command ${DESKTOPTOJSON_LOCATION} -i ${desktop} -o ${json})
     if(DESKTOP_TO_JSON_COMPAT_MODE)
       list(APPEND command -c)
     endif()
